app/lib/katello/qpid/connection.rb

```diff
@@ -49,17 +49,18 @@ def on_message(_delivery, message)
       end
 
       class Receiver < ::Qpid::Proton::MessagingHandler
-        def initialize(url, address, handler)
+        def initialize(url, connection_options, address, handler)
```

Comment:
The Sender method signature also needs an update -- https://github.com/Katello/katello/pull/9166/files#diff-d5d0f092b1171d0232fd04dc54b5b9f9dc095735857a4f8e7aecf0bd1f35dabbR7